types/index.d.ts

```diff
@@ -1,24 +1,25 @@
+import { ChartDatasetProperties, ChartType, Plugin, PluginOptions } from "chart.js";
 import { Options } from './options';
 
 declare module 'chart.js' {
-	interface ChartDataSets {
+	interface ChartDatasetProperties<TType extends ChartType, TData extends unknown[]> {
 		/**
 		 * Per dataset datalabels plugin options.
 		 * @since 0.1.0
 		 */
 		datalabels?: Options
 	}
 
-	interface ChartPluginsOptions {
-		/**
-		 * Per chart datalabels plugin options.
-		 * @since 0.1.0
-		 */
+  interface PluginOptions {
```

Comment:
Wrong indentation.

Reply:
Comment below still wrongly indented. I will try to fix it when merging.

Reply:
Automated linting fixed does wonders for this stuff...will add back in a separate PR